stylesheet/contact.css

```diff
@@ -0,0 +1,151 @@
+@import url(./main.css);
+
+.contact--banner-container {
+  background-image: url(../images/banners/contactBanner.jpg);
+
+  background-color: var(--color-background);
+  height: 20vw;
+  width: 99vw;
+
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.contact .contact--header {
+  background-color: var(--color-copyright-container);
+}
+
+.contact--banner-container {
+  display: flex;
+}
+
+.contact--banner-container > h1 {
+  color: var(--color-background);
+
+  text-transform: capitalize;
+  margin: auto;
+}
+
+.contact--details-wrapper {
+  display: flex;
+  flex-direction: row;
+
+  justify-content: space-around;
+
+  align-items: center;
+
+  padding: 5vw;

```

Comment:
padding in general se pune in unitate fixa (eg `px`) pt ca `vw` e relative la marimea paginii